### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by anandre**
Update methods and properties in the guide to correct v12 syntax.

Update Node required version.

### CODE REVIEW

File: guide/popular-topics/embeds.md [hunk 1]
```diff
@@ -222,15 +218,11 @@ If the images doesn't display inside the embed but outside of it, double check y
...
-
-::: warning
-You cannot just resend the received embed structure! It's a [MessageEmbed](https://discord.js.org/#/docs/main/stable/class/MessageEmbed) rather than a RichEmbed and contains circular references that prevent sending.
-:::
+To achieve this, you retrieve the embed from the messages embed array (`message.embeds`) and pass it to the MessageEmbed constructor. The constructed MessageEmbed can then be edited before sending it again.
 
 ```js
 const receivedEmbed = message.embeds[0];
-const exampleEmbed = new Discord.RichEmbed(receivedEmbed).setTitle('New title');
+const exampleEmbed = new Discord.MessageEmbed(receivedEmbed).setTitle('New title');
```
Comment (izexi) [inline guide/popular-topics/embeds.md]: `receivedEmbed` is already an instance of a MessageEmbed
```suggestion
const exampleEmbed = receivedEmbed.setTitle('New title');
```

File: guide/popular-topics/miscellaneous-examples.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ client.on('message', message => {
 	if (message.content === '!play') {
 		if (message.channel.type !== 'text') return;
 
-		const { voiceChannel } = message.member;
+		const { voiceChannel } = message.member.voice;
```
Comment (izexi) [inline guide/popular-topics/miscellaneous-examples.md]:
```suggestion
		const voiceChannel = message.member.voice.channel;
```
or `const { channel: voiceChannel }` if you wanted to maintain the destructing assignment

Comment (anandre) [reply guide/popular-topics/miscellaneous-examples.md]: Updated

File: guide/popular-topics/embeds.md (hunk 1 above)
Comment (anandre) [reply guide/popular-topics/embeds.md]: Per the discussion in discord, going to keep this as-is for consistency and to avoid edge cases.

**Pull request merged**